cs-utilities/header-definitions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<additionalHeaders>
+    <cs-actions-java-header>
+        <firstLine>/*******************************************************************************</firstLine>
+        <beforeEachLine> </beforeEachLine>
+        <endLine> *******************************************************************************/</endLine>
+        <afterEachLine> </afterEachLine>
+        <firstLineDetectionPattern>/[*]{79}</firstLineDetectionPattern>
```

Comment:
why 79 and not 119? :laughing:

Reply:
but it is written "<firstLineDetectionPattern>/[*]{79}</firstLineDetectionPattern> <lastLineDetectionPattern> [*]{79}/</lastLineDetectionPattern>" 😁